8.0.Dockerfile

```diff
@@ -0,0 +1 @@
+10.0.Dockerfile
```

Comment:
IMO the 8.0 should be the one containing the Dockerfile, with the older ones inheriting. This would align with what we will do when the 12.0 comes out, inheriting the old unless there is some new installation.